Docs/source/running_cpp/parameters.rst

```diff
@@ -1012,6 +1012,32 @@ Diagnostics and output
     Use a negative number to disable data dumping.
     This is ``-1`` (disabled) by default.
 
+    * ``<species_name>.random_fraction`` (`float`) optional
+        If provided ``<species_name>.uniform_stride = a``,
+        only `a` fraction of the particle data of this species will be dumped randomly,
+        i.e. if `rand() < a`, this particle will be dumped,
+        where `rand()` denotes a random number generator.
+        The value `a` provided should be between 0 and 1.
+
+    * ``<species_name>.uniform_stride`` (`int`) optional
+        If provided ``<species_name>.uniform_stride = n``,
+        every `n` particle of this species will be dumped,
+        i.e. if `particle ID % n == 0`, this particle will be dumped.
+        The value provided should be an integer greater than or equal to 0.
+
+    * ``<species_name>.plot_filter_function(t,x,y,z,ux,uy,uz)`` (`string`) optional
+        User can provide an expresion that describs the condition to filter
+        a portion of particles that will be dumped.
+        `t` represents the physical time in seconds during the simulation.
+        `x, y, z` represent particle positions in the unit of meter.
+        `ux, uy, uz` represent particle proper velocities
+        :math:`\gamma v`, where
+        :math:`\gamma` is the Lorentz factor,
+        :math:`v` is the particle velocity.
+        E.g. If provided `(x>0.0)*(uz<10.0)` only those particles located at
+        positions `x` greater than `0`, and those having velocity `uz` less than 10,
```

Comment:
```suggestion
        positions `x` greater than `0`, and those having momentum `uz` less than 10,
```

Reply:
I will use gamma*beta, so it is still velocity.